Dockerfile

```diff
@@ -1,2 +1,14 @@
-FROM ghcr.io/navikt/baseimages/temurin:21
-COPY /target/veilarbperson.jar app.jar
\ No newline at end of file
+FROM gcr.io/distroless/java21-debian12:nonroot
+LABEL org.opencontainers.image.source="https://github.com/navikt/veilarbperson"
+
+WORKDIR /app
+
+COPY /target/veilarbperson.jar app.jar
+
+USER nonroot
+
+ENV LANG='nb_NO.UTF-8' LC_ALL='nb_NO.UTF-8' TZ="Europe/Oslo"
+
+ENV TZ="Europe/Oslo"
+EXPOSE 8080
+CMD ["-jar", "app.jar"]
```

Comment:
Har sett at flere setter `-Xms<noe>` og  `-Xmx<noe>`, er det noe vi kanskje også burde gjøre? 🤔

Reply:
Jeg er enig i det fleste du peker på og tenker vi har ikke behov for å legge Label, User, WorkDir fordi noe vi definerer implicit og noen commander brukte jeg bare fordi de legger meta-data men de er ikke nødvendige. 
Om -Xms<noe> og -Xmx<noe>, tenker jeg at om de har definert noen sted for ex. i yaml file da er det kanskje ikke behov for å legge de igjen men en ting det jeg lurer på nå at i main-gcp.yaml file står det sånn `      
- name: Set up Java
        uses: actions/setup-java@v4
        with:
          java-version: 21
          distribution: 'temurin'
          cache: 'maven'`

her distribution er temurin og i dockerfile bytter vi det til gcr.io/distroless/java21-debian12 så trenger vi endre distribution i denne yaml filen eller la det være sånn?

Reply:
Det som står i workflowen handler bare om konteineren som bygger koden, så det skal så vidt jeg har skjønt ikke påvirkes av hva vi velger å kjøre koden på